docs/develop/local_dev.mdx

```diff
@@ -0,0 +1,188 @@
+    sudo mv restate-server /usr/local/bin/
+    ```
+
+    ```shell Linux-arm64
+    curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.aarch64-unknown-linux-musl.tar.gz && \
+    tar -xvf restate.aarch64-unknown-linux-musl.tar.gz && \
+    chmod +x restate restate-server && \
+    sudo mv restate /usr/local/bin/ && \
+    sudo mv restate-server /usr/local/bin/
+    ```
+</CH.Code>
+
+<Admonition type="note" title="Server and CLI configuration">
+    Have a look at the [CLI configuration docs](/operate/configuration/cli) or [Server configuration docs](/operate/configuration/server) for more configuration options.
+</Admonition>
+
+
+## Running Restate locally
+
+<CH.Spotlight className={"spotlight-medium"}>
+    Start the Restate Server:
+
+    ```shell
+    restate-server
+    ```
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+    Start the Restate Server and wipe ongoing invocations and K/V state.
+
+    ```shell
+    restate-server --wipe all
+    ```
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+    Check out all available commands and configuration options ([docs](/operate/configuration/server)):
+
+    ```shell
+    restate-server --help
+    ```
+</CH.Spotlight>
+
+## Useful dev CLI commands
+
+<CH.Spotlight className={"spotlight-medium"}>
+    Check if the CLI is installed correctly and can find the server:
+
+    ```shell
+    restate whoami
+    ```
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+    Register a new service deployment:
+
+    ```shell
+    restate deployments register localhost:9080
+    ```
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+
+    [Cancel](/operate/invocation#cancelling-invocations) a single invocation or a batch of invocations.
+    Use `--kill` to [kill](/operate/invocation#killing-invocations) the invocation.
+    To remove all invocations, restart the server with `--wipe all`.
+
+    ```shell
+    restate invocation cancel <INVOCATION_ID>
+    # also works with <SERVICE>/<SERVICE_KEY>/<HANDLER> or a subset of it
+    ```
+
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+
+    Purge completed invocations from the state:
+
+    ```shell
+    restate invocation purge <INVOCATION_ID>
+    # also works with <SERVICE>/<SERVICE_KEY>/<HANDLER> or a subset of it
+    ```
+
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+
+    Clear the K/V state of a Virtual Object or Workflows.
+    To clear all state, restart the server with `--wipe all`.
+
+    ```shell
+    restate kv clear <OBJECT_OR_WORKFLOW_NAME>
+    restate kv clear <OBJECT_OR_WORKFLOW_NAME>/<SERVICE_KEY>
+    ```
+
+</CH.Spotlight>
+<CH.Spotlight className={"spotlight-medium"}>
+
+    Execute a SQL query on the invocation or application state.
+    See [SQL introspection docs](/operate/introspection?interface=psql#inspecting-invocations) for example queries.
+
+    ```shell
+    restate sql --json "query"
```

Comment:
Remove the `--json` example here, if you don't need to pipe the `--json` output is hard to read.

Perhaps you can briefly mention an example where you show how to pipe `--json` output to some other command using jq, for example:

```
# Cancel all pending invocations to service "Greeter"
restate sql --json "SELECT id FROM sys_invocation WHERE target_service_name = 'Greeter' AND status = 'pending'" | jq .id | xargs restate invocation cancel --yes
```